proto/executor.proto

```diff
@@ -322,6 +327,18 @@ message TiFlashScanContext {
 	optional uint64 total_disagg_read_cache_hit_size = 12;
 	optional uint64 total_disagg_read_cache_miss_size = 13;
 	optional uint64 total_dmfile_rough_set_index_check_time_ms = 14;
+	optional uint64 num_columns = 15;
```

Comment:
* `num_columns` and `read_mode` maybe not that useful to be shown in the execution plan
* those `total_` prefix can be removed